components/logicbutton/stories/logicbutton.stories.js

```diff
@@ -34,19 +36,32 @@ export default {
 		isDisabled: false,
 	},
 	parameters: {
-		componentVersion: version,
+		componentVersion: version,	
+		docs: {
+			story: {
+				height: "auto",
+			},
+		},
 	},
 };
 
-export const Default = Template.bind({});
+export const Default = (args) => html`
+	${window.isChromatic() ? ChromaticLogicButtonGroup(args) : Template(args)}
+`;
 Default.args = {};
 
 export const Or = Template.bind({});
 Or.args = {
 	variant: "or"
 };
+Or.parameters = {
```

Comment:
It feels a little silly to have an "Or" story but not an "And" story (since it's technically the default), but in the interest of not rocking the boat with modes, it's a good idea to keep these stories named the way they are for now.

What do you think about maybe showing just "And" as the Default story, and then adding "Or" to the Disabled story? Then it could be organized on the Docs page like:
* Default
  * And only
* Disabled
  * And, Or group
* Or
  * Or only

Just wondering if maybe that would help the stories to feel a little more... logical 🥁

Reply:
Oh yeah! What if we did a similar thing to inline alert, where I actually have both? I can keep default in the sidenav, and then "And" is on the docs page with a note about it being the default? 

Also, I added the Or variant to the disabled story 🎉 https://github.com/adobe/spectrum-css/pull/2884/commits/c84c56f1c6b173a4fd2d659288dc93b165a64e5a

Reply:
I think that works! I left a note below with a small suggestion for the comment on the "And" story!